index.Rmd

```diff
@@ -261,23 +262,18 @@ as(MNDWI, "SpatialPixelsDataFrame") %>%
 
 I loaded required packages
 
-```{r, message=F, warning=F}
-library(threejs)
-```
 
 ##Results2
 I firstly prepared and loaded the data and then with the package called "streejs" I plot the exact locations of landing and mining sites on the Martian global map. I created a cvs file where I stored the latitudes and logitude of the future landing and mining sites selested and call that file in R.
 
 
 ```{r}
-library(threejs)
-library(csvread)
-the_martian <- read.csv("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landingMining_sites/the_martian.csv", comment.char="#")
+ the_martian <- read.csv("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landingMining_sites/the_martian.csv", comment.char="#")
```

Comment:
See my main comment. This raster is located on your personal machine, preventing myself or anyone else from running the code. It renders the rest of your code and analysis unable to run as well.